product_docs/docs/biganimal/release/free_trial/detail/create_a_cluster/create_cluster_portal.mdx

```diff
@@ -18,26 +18,25 @@ You should now find yourself at the [Create Cluster page](https://portal.biganim
 
 ### The Cluster Settings tab
 
-In the free trial, the range of cluster options is somewhat limited but should be more than sufficient for demonstrating the capabilities of BigAnimal. 
-
-1. Select your **Cluster Type**. Pick **Single Node**. If you're interested in learning more about the BigAnimal's high-availability deployment options, see [Cluster types](/biganimal/release/overview/02_high_availability).
-1. **Provider** is either AWS or Azure. BigAnimal works the same on either one, so choose the one you're familiar with.
+1. Select your **Cluster Type**. Pick **Single Node**. If you're interested in learning more about the BigAnimal's high-availability options, see [Cluster types](/biganimal/release/overview/02_high_availability).
+1. Select **BigAnimal** as your deployment type.
+1. **Provider** is AWS. 
 1. Use **Cluster Name** to describe the purpose of the cluster, for example, "Test for Advanced Server on AWS."
 1. **Password** is the admin password for the cluster, so pick a strong one! This password is independent of your EDB account.
 1. **Database Type** is one of:
 
-   - **Oracle Compatible** is powered by [EDB Postgres Advanced Server](/epas/latest/)
+   - **EDB Postgres Advanced Server** adds extended functionality to open-source PostgreSQL, see [EDB Postgres Advanced Server](/epas/latest/)
    - **PostgreSQL** for official community builds of standard PostgreSQL
 
-    If you're interested in EDB's enhancements to PostgreSQL for Oracle database compatibility, choose Advanced Server. Otherwise, choose PostgreSQL.
+    If you're interested in EDB's enhancements to PostgreSQL for Oracle database compatibility, choose EDB Postgres Advanced Server. Otherwise, choose PostgreSQL.
 1. **Postgres Version** allows you to select from among the supported versions for either database type.
 
 1. **Region** options depend on the provider you chose. Select the one closest to your application.
 
-1. For **Instance Type** and **Storage**, only one option is available for each field during the trial, so just choose that. The full sets of instance types and storage options are available in the BigAnimal paid subscriptions.
-1. **Network Connectivity** is restricted to a public IP for the trial, but you can restrict access to trusted networks by
-   selecting the **Use allowlists to restrict network access** check box and specifying one or more [CIDR blocks](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) 
-   to allow access from.
+1. For **Instance Type**, we recommend **Compute Optimized**, **AWS C5 instances**, **c5.large (2vCPU, 4GB RAM)** to get the most out of your free credits.
+1. For **Storage**, we recommend the **General Purpose** defaults.
```

Comment:
Let me know if we want to make this recommendation or if the storage option they pick doesn't matter in terms of efficient free credit usage.

Reply:
don't see that as necessary.  it's pretty intuitive from this point.

Reply:
After discussion with Jason we decided to not make any recommendations, but just refer users to the Create a cluster topic in the full documentation.